nativelib/src/main/scala/java/lang/ieee754tostring/ryu/RyuRoundingMode.scala

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 Ulf Adams
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Credits:
+//
+//    1) This work is a heavily modified derivation of the original work by
+//       Ulf Adams at URL: https://github.com/ulfjack/ryu.
+//       As such, it inherits the Apache license of the original work.
+//       Thank you Ulf Adams.
+//
+//    2) The original java sources were converted to a rough draft of
+//       scala code using the service at URL: javatoscala.com.
+//
+//       The raw conversion did not compile and contained bugs due
+//       to the handling of break and return statements, but it saved
+//       days, if not weeks, of effort.
+//
+//       Thank you javatoscala.com.
+//
+//    3) All additional work, including introduced bugs,  is an original
+//       contribution to Scala Native development.
+
+package java.lang.ieee754tostring.ryu
```

Comment:
Lets move this to `scala.scalanative.runtime.ryu` instead.

Reply:
I concur that moving to below scala.scalanative.runtime is an improvement.

I suggest keeping Ieee754 in the name, say scala.scalanative.ieee754.ryu
or DoubleFloat.ryu. This would make it easier for some poor developer, in
three month or a year out to find the code (and allow for alternate implementations).
Folks looking for stuff related to Double or Float at this level of detail have a sporting
 chance of knowing or googling Iee754.

I will implement what you suggest.  Thanks

Reply:
`scala.scalanative.ieee754.ryu` works as well.

Reply:
Done, voved to `scalanative/runtime/ieee754tostring/ryu/` and related files adjusted.

I like the fact that the trickery of `java.lang` code being in a non-obvious place is
eliminated.  Thank you.

Another kilometer down the pike.